curvefs/src/mds/fs_storage.cpp

```diff
@@ -163,6 +164,7 @@ FSStatusCode MemoryFsStorage::SetFsUsage(
     const std::string& fsName, const FsUsage& fsUsage) {
     WriteLockGuard writeLockGuard(fsUsedUsageLock_);
     fsUsageMap_[fsName] = fsUsage;
+    FsMetric::GetInstance().UpdateFsUsage(fsName, fsUsage);
```

Comment:
SetFsUsage may be more meaningful than UpdateFsUsage? depends on you

Reply:
> SetFsUsage may be more meaningful than UpdateFsUsage? depends on you

fix